drivers/wifi/nrf700x/osal/fw_if/umac_if/src/fmac_api_common.c

```diff
@@ -514,6 +518,58 @@ enum wifi_nrf_status wifi_nrf_fmac_rf_params_get(struct wifi_nrf_fmac_dev_ctx *f
 
 	ft_prog_ver = (ft_prog_ver & FT_PROG_VER_MASK) >> 16;
 
+#if defined(CONFIG_BOARD_NRF7002DK_NRF7001_NRF5340_CPUAPP) || \
+	defined(CONFIG_BOARD_NRF7002DK_NRF5340_CPUAPP) || \
+	defined(CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPP)
+	if (tx_pwr_ceil_params->rf_tx_pwr_ceil_params_override) {
+		if (ft_prog_ver == 1) {
+			backoff_2g_dsss = 0;
```

Comment:
Add a TODO that explains why these are all zeroes, if you intend to fill this with proper values before merging then its fine.